db/change_cache.go

```diff
@@ -344,7 +344,10 @@ func (c *changeCache) DocChanged(event sgbucket.TapEvent) {
 					rawBody = nil
 				}
 				db := Database{DatabaseContext: c.context, user: nil}
-				db.ImportDoc(docID, rawBody, isDelete)
+				err := db.ImportDoc(docID, rawBody, isDelete)
+				if err != nil {
+					base.Warn("Error importing doc %q: %v", docID, err)
```

Comment:
Can you make this message more verbose, in particular to answer the question on what are the implications of this error.  For example: "Error importing doc.  The changes made by the SDK to the Couchbase bucket will not be visible in Sync Gateway."